public/video-ui/src/util/getComposerData.js

```diff
@@ -1,107 +1,26 @@
-  const coreFields = [
-    {
-      name: 'headline',
-      value: video.title,
-      belongsTo: 'fields',
-      isFreeText: true
-    },
-    {
-      name: 'standfirst',
-      value: video.description ? video.description : null,
-      belongsTo: 'fields',
-      isFreeText: true
-    },
-    {
-      name: 'trailText',
-      value: video.trailText ? video.trailText : null,
-      belongsTo: 'fields',
-      isFreeText: true
-    },
-    {
-      name: 'linkText',
-      value: video.title,
-      belongsTo: 'fields'
-    },
-    {
-      name: 'sensitive',
-      value: asBooleanString(video.sensitive),
-      belongsTo: 'settings'
-    },
-    {
-      name: 'legallySensitive',
-      value: asBooleanString(video.legallySensitive),
-      belongsTo: 'settings'
-    },
-    {
-      name: 'blockAds',
-      value: asBooleanString(video.blockAds),
-      belongsTo: 'settings'
-    },
-    {
-      name: 'commentable',
-      value: asBooleanString(video.composerCommentsEnabled),
-      belongsTo: 'settings'
-    },
-    {
-      name: 'relatedContentOff',
-      value: asBooleanString(video.suppressRelatedContent),
-      belongsTo: 'settings'
-    },
-    {
-      name: 'seoOptimised',
-      value: asBooleanString(video.optimisedForWeb),
-      belongsTo: 'toolSettings'
-    },
-    {
-      name: 'byline',
-      value: video.byline.join('|'),
-      belongsTo: 'atom'
-    },
-    {
-      name: 'commissioningDesks',
-      value: video.commissioningDesks.join('|'),
-      belongsTo: 'atom'
-    },
-    {
-      name: 'keywords',
-      value: video.keywords.join('|'),
-      belongsTo: 'atom'
-    },
-    {
-      name: 'thumbnail',
-      value: (video.trailImage && video.trailImage.assets.length > 0)
-        ? parseComposerDataFromImage(video.trailImage, video.trailText)
-        : null,
-      belongsTo: 'thumbnail'
-    }
-  ];
-
   const isTrainingMode = getStore().getState().config.isTrainingMode;
-
-  // block training content being published
-  const embargoedIndefinately = {
-    name: 'embargoedIndefinitely',
-    value: asBooleanString(true),
-    belongsTo: 'settings'
+  return {
+    headline: video.title,
+    standfirst: video.description ? video.description : null,
+    trailText: video.trailText ? video.trailText : null,
+    linkText: video.title,
+    sensitive: video.sensitive,
+    legallySensitive: video.legallySensitive,
+    blockAds: video.blockAds,
+    commentable: video.composerCommentsEnabled,
+    relatedContentOff: video.suppressRelatedContent,
+    seoOptimised: video.optimisedForWeb,
+    commissioningDesks: video.commissioningDesks.join('|'),
+    byline: video.byline.join('|'),
+    keywords: video.keywords.join('|'),
+    thumbnail: video.trailImage && video.trailImage.assets.length > 0
+      ? parseComposerDataFromImage(video.trailImage, video.trailText)
+      : null,
+    expiryDate: video.expiryDate,
+    embargoedIndefinitely: isTrainingMode ? true : false
```

Comment:
don't need the ternary anymore

Reply:
why not?

Reply:
we can do `embargoedIndefinitely: isTrainingMode` instead